### DISCUSSION TITLE
Simulation Mechanism

### DISCUSSION THREAD

**Discussion by sandeepgowdan** [gaynorr/AlphaSimR#180] [Q&A]
1. Simulation Mechanism: In your paper, you discuss how each trait is controlled by multiple Quantitative Trait Loci (QTLs), and these QTLs vary in their location across different regions in various crops. Could you please elaborate on how the simulation accounts for these complexities, particularly in terms of  the demographic history of each species?
2. Additionally, I am curious about how the simulation mechanism addresses the challenge of developing founder populations using Runmacs, where different species may have QTLs located in diverse regions. Could you provide further clarification on how the simulation manages this aspect? Before simulatioon should we have demogarphic  history of the sepcies present in the ALphasimR

**Comment by gregorgorjanc** [comment]
> * Simulation Mechanism: In your paper, you discuss how each trait is controlled by multiple Quantitative Trait Loci (QTLs), and these QTLs vary in their location across different regions in various crops. Could you please elaborate on how the simulation accounts for these complexities, particularly in terms of  the demographic history of each species?

We simulate multiple QTL at random positions. This is the best we can do in the absence of other knowledge. If you know QTL locations (and maybe even their effects) then you can import this information into simulation as we mention and show in the pre-print material https://www.biorxiv.org/content/10.1101/2023.12.30.573724v1.full.pdf

**Comment by gregorgorjanc** [comment]
> 2\. Additionally, I am curious about how the simulation mechanism addresses the challenge of developing founder populations using Runmacs, where different species may have QTLs located in diverse regions. Could you provide further clarification on how the simulation manages this aspect? Before simulatioon should we have demogarphic  history of the sepcies present in the ALphasimR

As above, if we know this information, we can import it. We often don't, or have minimal information. A user can choose QTL themselves and import that into the AlphaSimR SP object.

The background coalescent simulator via MaCS (as called by `runMacs()`) is neutral, so QTL do not play into this.

**Comment by sandeepgowdan** [comment]
Thank you sir
If we simulate 100 SegSites for each chromosome for two different population of genotypes, one population having high heritability and high variance and other population having low heritability and low variance then does the genetic gain purely depends on the heritability and variances

**Comment by gregorgorjanc** [comment]
I am not sure what you are trying to achieve @sandeepgowdan so these questions are probably not most informative;) I assume that whatever the number of QTL you will pick, you likely will not have massively different error variance (and hence heritability) differences between the populations. However, if this is the case, then the population with lower h2 will have lower genetic gain. You can convince yourself by running a simulation;)

**Comment by sandeepgowdan** [comment]
Ok sir thank you
